src/cdk/tree/control/nested-tree-control.spec.ts

```diff
@@ -196,6 +197,24 @@ describe('CdkNestedTreeControl', () => {
       });
     });
   });
+
+  it('maintains node expansion state based on trackBy function, if provided', () => {
+    const treeControl = new NestedTreeControl<TestData, string>(getChildren);
```

Comment:
Instead of adding the trackBy directly, can you utilize the new constructor `options` arg here? That would be closer to the implementation we expect users to have

Reply:
Sounds good, updated this test to use the `options` arg. I initially was copying what was done for FlatTreeControl, should I update that test to use FlatTreeControl's `options` as well? https://github.com/angular/components/blob/b089abe60700a95618ad15c94f289547f6b3acf9/src/cdk/tree/control/flat-tree-control.spec.ts#L150